spec/models/post_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  # Create post
+
+  subject { Post.create(author: User.new(name: 'morris'), title: 'test_title', text: 'test text') }
```

Comment:
- [ ] Most of your Post spec tests are failing because you are not creating the User. Also `User.new` does not really create a user, it only instantiates the model object. You still need to call `#save` to actually save the method in the database. Your user model requires `posts_counter` validation. You can fix this by changing this something like this:
  
  ```
  user = User.create(name: "Morris", posts_counter: 0)
  subject { Post.create(author_id: user.id, title: 'test_title', text: 'test text', comments_counter: 0, likes_counter: 0) }
  ```